src/UserAdmin/src/Smart.FA.Catalog.Web/Validators/CreateTrainingRequestValidator.cs

```diff
@@ -10,24 +11,30 @@ namespace Smart.FA.Catalog.Web.Validators;
 /// </summary>
 public class CreateTrainingRequestValidator : AbstractValidator<CreateTrainingViewModel>
 {
-    public CreateTrainingRequestValidator(IStringLocalizer<CatalogResources> localizer)
+    public CreateTrainingRequestValidator(IStringLocalizer<CatalogResources> localizer, IUserIdentity userIdentity)
     {
         RuleFor(viewModel => viewModel.Title)
             .NotEmpty()
             .MaximumLength(500)
             .WithMessage(CatalogResources.TrainingTitleIsRequired);
 
-            RuleFor(viewModel => viewModel.Methodology)
-                .MaximumLength(1000)
-                .WithMessage(CatalogResources.Max1000Characters);
+        RuleFor(viewModel => viewModel.Methodology)
+            .MaximumLength(1000)
+            .WithMessage(CatalogResources.Max1000Characters);
+
+        RuleFor(viewModel => viewModel.Goal)
+            .MaximumLength(1000)
+            .WithMessage(CatalogResources.Max1000Characters);
 
-            RuleFor(viewModel => viewModel.Goal)
-                .MaximumLength(1000)
-                .WithMessage(CatalogResources.Max1000Characters);
+        RuleFor(viewModel => viewModel.PracticalModalities)
+            .MaximumLength(1000)
+            .WithMessage(CatalogResources.Max1000Characters);
+
+        RuleFor(viewModel => viewModel.PracticalModalities)
+            .MaximumLength(1000)
+            .WithMessage(CatalogResources.Max1000Characters);
 
-            RuleFor(viewModel => viewModel.PracticalModalities)
-                .MaximumLength(1000)
-                .WithMessage(CatalogResources.Max1000Characters);
+        When(_ => !userIdentity.IsSuperUser, () => RuleFor(viewModel => viewModel.IsGivenBySmart).NotEqual(true).WithMessage(CatalogResources.SuperUserPermissionToSetSmartTrainingType));
```

Comment:
```suggestion
        RuleFor(viewModel => viewModel.IsGivenBySmart)
            .Must(BeSuperUser)
            .When(GivenBySmartIsMarked)
            .WithMessage(CatalogResources.SuperUserPermissionToSetSmartTrainingType);      
```
It is always a good practice to use method groups when doing fluent chaining.